pkg/iptables/fake/iptables.go

```diff
@@ -156,6 +156,18 @@ func (i *IPTables) ClearChain(table, chain string) error {
 	return nil
 }
 
+func (i *IPTables) ChainExists(table, chain string) (bool, error) {
```

Comment:
Not sure if the error return value is needed for  ChainExists function, cause ChainExists always returns error ==nil, but maybe we should keep this way for consistency

Reply:
This is a fake implementation, so we have to keep the API in sync for consistency.